.cargo/config.toml

```diff
@@ -1,3 +1,7 @@
 [build]
 # v0 mangling scheme provides more detailed backtraces around closures
 rustflags = ["-C", "symbol-mangling-version=v0"]
+
+# Shows private items in the generated documentation when running `cargo doc`
+# Makes rustdoc more useful for understanding the codebase, not just the public api surface
+rustdocflags = ["--document-private-items"]
```

Comment:
I'm not positive we want this flag here.

I added it because there are many items that are in private modules that have documentation that's useful for understanding the codebase. 

An example of this would be the module documentation for `editor::display_map`.  The existing documentation referenced many private types, such as `InlayMap`, making `cargo doc` not useful for understanding this part of the codebase.

Reply:
I think the question we should ask is "will we publish these crates to crates.io and docs.rs?"  

If the answer is yes, we should disable this flag, and adjust our documentation to only reference public items.

If the answer is no, then `cargo doc` is an incredibly useful tool for understanding the local codebase, and I feel like this change is warranted.

Reply:
I don't think we're planning on publishing these docs anywhere, they're entirely internal and won't be exposed otherwise.

Reply:
We're probably not going to be publishing these as docs anytime soon. That said, let's let folks pick which documentation they prefer. This should be removed.

Reply:
I don't think we should be including private items in rustdoc, as it makes it less clear what the public interface of a given crate is.

Reply:
Makes sense, I've removed the flag!